docs/contributing.md

```diff
@@ -39,7 +158,7 @@ Tests are shell scripts in `tests/scripts` with names like `test_*.sh`.
 They should should focus on the cli operation and are not a replacement for pulp integration tests;
 i.e. make sure the cli translates to the right api calls, but do not care about pulp internals.
 
-### Running Tests
+## Running Tests
```

Comment:
This makes me think, we should state that we require tests for features in the conventions section (It's better to set the expectations strait upfront.).
Or maybe we add a PR checklist.

Reply:
I added a line but I agree with a PR checklist, but perhaps that can be an additional PR that is out of the scope of this rearrangement PR. 
I was wondering if an example test for a feature would be helpful also. 
But perhaps anyone contributing to the CLI would not need such handholding. I always tend to lean towards highest level of handholding we can offer :-)

Reply:
+1 to followup PR's.
Handholding can include that we run shell scripts located in the `tests/scripts` directory.